.devcontainer/build.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+HERE="$(dirname "$0")"
+CHIP_ROOT="$(realpath "$HERE"/..)"
+BUILD_TAG=$(head -n 1 "$CHIP_ROOT"/integrations/docker/images/chip-build/version | sed 's/\s.*$//' | awk '{print $1}')
```

Comment:
This may not be up to date. When a new SDK is used, we sometimes have flows like:

1) update docker image and version to contain a new SDK (code is not yet new SDK compatible)
2) update code to be compatible with the new SDK and update CI workflows and docker image reference version


In between 1) and 2) the latest docker image version cannot compile the code.

Could we grab the version from somewhere else like some of the CI .github/workflow images?

Reply:
Looking at the script, where is $BUILD_TAG used?

Reply:
manual update seems fine. This is what we had until now and I am unsure how to make it any cleaner without optimizing for "temporary updates" while potentially breaking regular use. I think making the regular use more stable is the correct path.